code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -322,8 +322,7 @@ meteor_act
 					return
 
 		var/dtype = O.damtype
-		var/throw_damage = O.throwforce * (speed / THROWFORCE_SPEED_DIVISOR)
-
+		var/throw_damage = O.throwforce
```

Comment:
What effect does it have on throw damage? With examples, please.

Reply:
speed / THROWFORCE_SPEED_DIVISOR (almost) always results in 1, because in proc 
(/mob/living/carbon/human/hitby(atom/movable/AM as mob|obj,__var/speed = THROWFORCE_SPEED_DIVISOR__))
speed equals divisor. No idea why is this here, honestly.

Reply:
To my knowledge speed is defined in the item, while divisor is a global define. Usually fairly important, as it determines the damage by a large margin. Handyman knows more about it than I do, however, so I would ask him about it.

It also has to do with catching knives.